README.md

```diff
@@ -13,21 +13,28 @@ and can be used with any editor that supports the protocol, to offer good suppor
 
 Features
 --------------
-* As you type reporting of parsing and compilation errors
+* Supports compiling projects from Java 1.5 through 15
```

Comment:
Since we hold off Java 16 support in 1.0 release, i don't put 16 support to the features list yet.